active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+        IsCloseButtonVisible="False">
+    </controls:InfoBar>
+</StackPanel>
+```
+![Mockup of an InfoBar in a Critical state with no close button](images/Critical_IconNoClose.png)
+
+## Custom styling: status color and icon
+Outside of the pre-defined severity levels, the Background and IconSource properties can be set to customize the icon and background color. The info bar will retain the assistive technology settings of the severity defined, or default if none was defined.
+
+A custom background color can be set via the standard Background property and will override the color set by Severity. Please keep in mind content readability and accessibility when setting your own color.
+
+A custom icon can be set via the IconSource property left of the Title and Message in the InfoBar. By default, an icon will be visible. This icon can be removed by setting the IsIconVisible property to false. For custom icons, the recommended icon size is 20px.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="NewPhotosNotification"
+        Title="New Photos From Cloud"
+        Message="Sync your latest photos to the Cloud"
+        ActionButtonContent="Upload">
+        <controls:InfoBar.Background>
+            <SolidColorBrush Color="#FF557EB9"/>
+        </controls:InfoBar.Background>
+        <controls:InfoBar.IconSource>
+            <SymbolIconSource Symbol="Camera" />
+        </controls:InfoBar.IconSource>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar in the top of the content area. The notification's title is "New Photos From Cloud" and it's message is "Sync your latest photos to the Cloud". It has a bright green accent color and a "Camera" icon.](images/Custom_IconColor.png)
+
+## Add buttons
+By default, an 'X' close button will appear as the right most component in the bar.
+
+An additional action button can be added by setting the ActionButtonContent and ActionButtonCommand properties. There is also built-in support for a single hyperlink button to ensure text contrast remains accessible with the various background colors. We recommend that only a single action button or hyperlink button is set. Outside of these, additional action buttons and hyperlinks can be added via custom content.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+        Severity="Warning"
+        Title="Error while saving"
+        Message="Your document was unable to be saved."
+        ActionButtonContent="Try again"
+        ActionButtonCommand="SaveDocuments">
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar with a single line message and an action button](images/Warning_Button.png)
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="ConnectionErrorNotification"
+        Title="Error while saving"
+        Message="Lorem ipsum long message">
+        <controls:InfoBar.HyperlinkButtonContent>
```

Comment:
It's odd that I can provide an actual HyperlinkButton object but not a Button object.